src/AlternativeHierarchies/HierarchyNotes/HierarchyNotes.ts

```diff
@@ -22,15 +22,16 @@ export async function getHierarchyNoteItems(plugin: BCPlugin, file: TFile) {
   const problemFields: string[] = [];
 
   const upFields = getFields(userHiers, "up");
+  const nextFields = getFields( userHiers , "next")
```

Comment:
What are those nextFields for?

Reply:
It seems like you're also changing something in hierarchy notes with there being next and previous links which is not explained in the PR.

Reply:
oops. I forget remove this. I was trying to implement the feature I was explaining in #327. But it is useless in this two context